x-pack/plugins/observability/public/pages/overview/panels/cpu_panel.ts

```diff
@@ -0,0 +1,165 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { ObsPanel } from './types';
+
+export const cpuPanel: ObsPanel = async (observabilityDataViews, id, coordinates) => {
+  const dataView = await observabilityDataViews.getIndexPattern('infra_metrics');
+
+  return {
+    gridData: {
+      ...coordinates,
+      i: id.toString(),
+    },
+    type: 'lens',
+    explicitInput: {
+      id: id.toString(),
+      attributes: getLensAttributes(dataView),
+    },
+  };
+};
+
+function getLensAttributes(dataView: DataView) {
+  return {
+    title: 'CPU by Host',
```

Comment:
How easy it has been to get this config? Is there some documentation about it? Just to understand how easy it would be to maintain them.

Reply:
what I did was to create a lens visualization in Kibana and later copy/paste its saved object. Relatively easy but not straightforward.

Moving forward we can either make helpers that generate sane defaults, or lobby the Lens team to make those helpers for us :D

Reply:
@afgomez we already have these utilities in exploratory view, i think those can be adapted to generate lens attributes on the fly. 

https://github.com/elastic/kibana/blob/main/x-pack/plugins/observability/public/components/shared/exploratory_view/configurations/lens_attributes.ts